scripts/sql2html.pl

```diff
@@ -49,6 +52,7 @@
     'format_headers=i' => \$format_headers,
     'sort_headers=i'   => \$sort_headers,
     'sort_tables=i'    => \$sort_tables,
+    'out_diagram_dir=s'=> \$out_diagram_dir,
```

Comment:
Could you add a description of this new option in the "usage()" method, please ?

Reply:
Good point. I've added it